tfx/experimental/distributed_inference/graphdef_experiments/subgraph_partitioning/beam_pipeline_test.py

```diff
@@ -25,7 +25,11 @@
 from tfx.experimental.distributed_inference.graphdef_experiments.subgraph_partitioning import create_complex_graph
 from tfx.experimental.distributed_inference.graphdef_experiments.subgraph_partitioning import graph_partition
 
-tf.compat.v1.enable_eager_execution()  # Re-enable eager mode
+@pytest.fixture
+def enable_eager_execution_at_test_end():
+  yield
+  # Re-enable eager mode that create_complex_graph diabled.
+  tf.compat.v1.enable_eager_execution()
```

Comment:
I think it's preferable for this to be located in [`conftest.py`](https://github.com/tensorflow/tfx/blob/master/conftest.py) so that it needs to only appear once in the code.

Reply:
It also appears to me that you've defined the fixture, but haven't told pytest to use it. You can do so by either the traditional method of [requesting (adding as a method argument)](https://docs.pytest.org/en/stable/how-to/fixtures.html#quick-example), or with a [`usefixtures` mark](https://docs.pytest.org/en/stable/how-to/fixtures.html#use-fixtures-in-classes-and-modules-with-usefixtures).

Reply:
I discovered that enable_eager_execution() is not compatible with this approach. All tests failed with the following error message: 'ValueError: tf.enable_eager_execution must be called at program startup.' 

Since tests under experimental folders passed without the enable_eager_execution() call, I just cleaned up the pytest.fixture code.